src/harness/testcases/WINNF_FT_S_BPR_testcase.py

```diff
@@ -291,6 +291,7 @@ def test_WINNF_FT_S_BPR_2(self, config_filename):
           closest_point_lat,
           closest_point_lon,
           closest_point_height,
+          reliability=0.5,
```

Comment:
Do these args match

propagation = wf_itm.CalcItmPropagationLoss(
          cbsd_lat,
          cbsd_lon,
          cbsd_height,
          closest_point_lat,
          closest_point_lon,
          closest_point_height,
          reliability=0.5,
          cbsd_indoor=is_cbsd_indoor,
          freq_mhz=freq_mhz)

def CalcItmPropagationLoss(lat_cbsd, lon_cbsd, height_cbsd,
                           lat_rx, lon_rx, height_rx,
                           cbsd_indoor=False,
                           reliability=0.5,
                           freq_mhz=3625.,
                           its_elev=None,
                           is_height_cbsd_amsl=False):

Reply:
It does not make a difference because we pass them as keyword arguments.
In fact, in Python, when an keyword argument is specified in the function prototype with a default value, it is best practice to always call the routine with keywords arguments (reliability=0.5, etc..).
Hence these arguments order can be rearranged later if you wish, without changing the client code.
So you dont have to worry about the order.

Reply:
Thanks :)